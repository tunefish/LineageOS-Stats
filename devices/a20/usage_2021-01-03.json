{
  "17.1-20200715-unofficial-tissot": 1455,
  "17.1-20200821-unofficial-z3": 963,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.diust.20201222.182702 test-keys": 3,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.igor.20200914.210059 test-keys": 2
}