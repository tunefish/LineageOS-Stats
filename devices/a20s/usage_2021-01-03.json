{
  "17.1-20200715-unofficial-tissot": 1503,
  "17.1-20200821-unofficial-z3": 979
}