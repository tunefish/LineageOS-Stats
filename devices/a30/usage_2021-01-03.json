{
  "17.1-20200715-unofficial-tissot": 2,
  "17.1-20200821-unofficial-z3": 30,
  "18.0-20201017_054056-unofficial-a30": 4,
  "18.0-20201017_104949-unofficial-a30": 141,
  "18.0-20201023_162226-unofficial-a30": 75,
  "18.0-20201101_142030-unofficial-a30": 1,
  "18.0-20201102_092137-unofficial-a30": 4,
  "18.0-20201102_152557-unofficial-a30": 142,
  "18.0-20201118_095214-unofficial-a30": 82,
  "18.0-20201123_110028-unofficial-samarv-121-a30": 2,
  "18.0-20201201_161908-unofficial-samarv-121-a30": 2,
  "18.0-20201204_050642-unofficial-samarv-121-a30": 9,
  "18.0-20201205_095022-unofficial-samarv-121-a30": 3,
  "18.0-20201206_134506-unofficial-samarv-121-a30": 21,
  "18.0-20201206_210235-unofficial-samarv-121-a30": 75,
  "18.1-20201223_053153-unofficial-samarv-121-a30": 22,
  "18.1-20201229_131728-unofficial-3037829-a30": 13,
  "3.r": 172,
  "3.r.beta-2": 2,
  "3.r.beta3": 148,
  "4.r": 32,
  "built.with.erfangsi.tools": 4,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.diust.20201222.182702 test-keys": 16,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.igor.20201013.190840 test-keys": 3
}