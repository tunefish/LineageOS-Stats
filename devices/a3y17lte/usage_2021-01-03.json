{
  "14.1-20180205-unofficial-a3y17lte": 3,
  "14.1-20180614-unofficial-j5xnlte": 9559,
  "15.1-20180227-unofficial-a3y17lte": 1,
  "15.1-20180306-unofficial-a3y17lte": 4,
  "15.1-20180416-unofficial-a3y17lte": 1,
  "15.1-20180517-unofficial-a3y17lte": 7,
  "15.1-20180821-unofficial-a3y17lte": 6,
  "15.1-20181205-unofficial-a3y17lte": 262,
  "15.1-20201128-unofficial-herolte": 6,
  "16.0-20181215-unofficial-a3y17lte": 52,
  "16.0-20190307-unofficial-a3y17lte": 8,
  "16.0-20190330-unofficial-a3y17lte": 12,
  "16.0-20190410-unofficial-a3y17lte": 18,
  "16.0-20190419-unofficial-a3y17lte": 2,
  "16.0-20190426-unofficial-a3y17lte": 4,
  "16.0-20190430-unofficial-a3y17lte": 2,
  "16.0-20190507-unofficial-a3y17lte": 1,
  "16.0-20190510-unofficial-a3y17lte": 3,
  "16.0-20190517-unofficial-a3y17lte": 3,
  "16.0-20190522-unofficial-a3y17lte": 7,
  "16.0-20190529-unofficial-a3y17lte": 4,
  "16.0-20190611-unofficial-a3y17lte": 4,
  "16.0-20190623-unofficial-a3y17lte": 73,
  "16.0-20190824-unofficial-a3y17lte": 1,
  "16.0-20190830-unofficial-a3y17lte": 10,
  "16.0-20190908-unofficial-a3y17lte": 640,
  "16.0-20190926-unofficial-a3y17lte": 2,
  "16.0-20191008-unofficial-a3y17lte": 72,
  "16.0-20191126-unofficial-a3y17lte": 3,
  "16.0-20200621-unofficial-a3y17lte": 7,
  "17.1-20200119-unofficial-a3y17lte": 1,
  "17.1-20200124-unofficial-a3y17lte": 1,
  "17.1-20200125-unofficial-a3y17lte": 4,
  "17.1-20200210-unofficial-a3y17lte": 1,
  "17.1-20200221-unofficial-a3y17lte": 10,
  "17.1-20200403-unofficial-a3y17lte": 49,
  "17.1-20200613-unofficial-a3y17lte": 11,
  "17.1-20200713-unofficial-a3y17lte": 30,
  "17.1-20200715-unofficial-tissot": 7,
  "17.1-20200716-unofficial-herolte": 2,
  "17.1-20200821-unofficial-z3": 13,
  "17.1-20200825-unofficial-a3y17lte": 8,
  "17.1-20200830-unofficial-a3y17lte": 671,
  "17.1-20201009-unofficial-a3y17lte": 1,
  "17.1-20201016-unofficial-a3y17lte": 1,
  "17.1-20201024-unofficial-a3y17lte": 1,
  "17.1-20201109-unofficial-a3y17lte": 3,
  "17.1-20201112-unofficial-a3y17lte": 7,
  "17.1-20201112-unofficial-ef63": 1,
  "17.1-20201113-unofficial-tissot": 5,
  "17.1-20201114-unofficial-a3y17lte": 1,
  "17.1-20201116-unofficial-z3": 2,
  "17.1-20201222-unofficial-a3y17lte": 1,
  "18.0-20201028-unofficial-a3y17lte": 1
}