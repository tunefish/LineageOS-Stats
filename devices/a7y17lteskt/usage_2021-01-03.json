{
  "17.1-20200715-unofficial-tissot": 1468,
  "17.1-20200821-unofficial-z3": 970
}