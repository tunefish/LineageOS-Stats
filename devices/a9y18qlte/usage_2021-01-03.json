{
  "17.1-20200715-unofficial-tissot": 2871,
  "17.1-20200821-unofficial-z3": 1822,
  "17.1-20201015-unofficial-starlte": 1
}