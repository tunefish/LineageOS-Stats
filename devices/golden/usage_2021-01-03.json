{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 99,
  "13.0-20170224-unofficial-golden": 6,
  "13.0-20170302-unofficial-golden": 8,
  "13.0-20170310-unofficial-golden": 25,
  "13.0-20170328-unofficial-golden": 818,
  "13.0-20170420-unofficial-golden": 54,
  "14.1-20170122-unofficial-golden": 270,
  "14.1-20170318-unofficial-golden": 5,
  "14.1-20170402-unofficial-golden": 12,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170503-unofficial-golden": 1,
  "14.1-20170507-unofficial-golden": 17,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 512,
  "14.1-20170610-unofficial-golden": 10,
  "14.1-20170628-unofficial-golden": 806,
  "14.1-20170724-unofficial-golden": 47,
  "14.1-20170817-unofficial-golden": 34,
  "14.1-20170920-unofficial-golden": 8,
  "14.1-20171011-unofficial-golden": 74,
  "14.1-20171215_192311-official-golden": 269,
  "14.1-20171215_192311-unofficial-golden": 965,
  "14.1-20180109_002536-unofficial-golden": 208,
  "14.1-20181014-unofficial-golden": 3,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "17.1-20201113-unofficial-tissot": 2,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}